source/classes.tex

```diff
@@ -6817,43 +6903,45 @@
 \end{codeblock}
 
 \item
-Otherwise, if \tcode{R} is \tcode{partial_ordering}, then
+Otherwise, (when \tcode{R} is \tcode{partial_ordering}),
 \begin{codeblock}
 a == b ? partial_ordering::equivalent :
 a < b  ? partial_ordering::less :
 b < a  ? partial_ordering::greater :
          partial_ordering::unordered
 \end{codeblock}
-
-\item
-Otherwise, the synthesized three-way comparison is not defined.
 \end{itemize}
+
 \begin{note}
 A synthesized three-way comparison may be ill-formed
-if overload resolution finds usable functions
+if overload resolution finds usable candidates
 that do not otherwise meet the requirements implied by the defined expression.
 \end{note}
 
 \pnum
 Let \tcode{R} be the declared return type of
-a defaulted three-way comparison operator function.
-Given an expanded list of subobjects for an object \tcode{x} of type \tcode{C},
-let $\tcode{R}_i$ be
-the type of the expression $\tcode{x}_i$ \tcode{<=>} $\tcode{x}_i$, or
-\tcode{void} if overload resolution applied to that expression
-does not find a usable function.
+a defaulted three-way comparison operator function, and
+let $\tcode{x}_i$ be the elements of
+the expanded list of subobjects for
+an object \tcode{x} of type \tcode{C}.
+
 \begin{itemize}
 \item
-If \tcode{R} is \tcode{auto},
-then the return type is deduced as
+If \tcode{R} is \tcode{auto}, then
+let $\tcode{\cv{}}_i$ $\tcode{R}_i$ be
```

Comment:
This should probably go into a single math env. (Avoiding a line break in between sounds like a good idea.)